test/integration/framework/master_utils.go

```diff
@@ -344,7 +343,6 @@ func NewMasterConfig() *master.Config {
 	kubeVersion := version.Get()
 	genericConfig.Version = &kubeVersion
 	genericConfig.Authorizer = authorizerfactory.NewAlwaysAllowAuthorizer()
-	genericConfig.AdmissionControl = admit.NewAlwaysAdmit()
```

Comment:
Revert all these… they should continue to function

Reply:
ok